vars/cloudFoundryDeploy.groovy

```diff
@@ -222,6 +227,15 @@ void call(Map parameters = [:]) {
             .withMandatoryProperty('cloudFoundry/credentialsId')
             .use()
 
+        if (config.useGoStep == true) {
```

Comment:
Not sure if it's necessary but you could also add a test case for the feature toggle to make sure only one or the other is executed.